tests/test_pay.py

```diff
@@ -2564,11 +2564,11 @@ def serialize_payload(n):
         assert(e.error['data']['raw_message'] == "400f00000000000003e80000006c")
 
 
-@unittest.skipIf(not DEVELOPER, "needs DEVELOPER=1")
+@unittest.skipIf(not DEVELOPER, "needs dev-disconnect, dev-no-htlc-timeout")
 @unittest.skipIf(not EXPERIMENTAL_FEATURES, "needs parallel_id support")
 def test_partial_payment(node_factory, bitcoind, executor):
     # We want to test two payments at the same time, before we send commit
-    l1, l2, l3, l4 = node_factory.get_nodes(4, [{}] + [{'disconnect': ['=WIRE_UPDATE_ADD_HTLC-nocommit']}] * 2 + [{}])
+    l1, l2, l3, l4 = node_factory.get_nodes(4, [{}] + [{'disconnect': ['=WIRE_UPDATE_ADD_HTLC-nocommit'], 'dev-no-htlc-timeout': None}] * 2 + [{}])
```

Comment:
The holding of HTLCs is necessary so the `waitsendpay` isn't just resolved internally below if the response comes back in time?

Reply:
Yes, and because this test was implemented before mpp, so I wanted to test the *send* side allowed two payments in parallel.